cli/cli_mesh_peers.go

```diff
@@ -624,6 +628,72 @@ func (c *cmd) renameMeshnetPeer(peer *pb.Peer, nickname string) error {
 	return nil
 }
 
+func (c *cmd) MeshSetMachineNickname(ctx *cli.Context) error {
+	if ctx.NArg() != 1 {
+		// needed nickname only
+		return argsCountError(ctx)
+	}
+	nickname := ctx.Args().First()
+	resp, err := c.meshClient.ChangeMachineNickname(context.Background(), &pb.ChangeMachineNicknameRequest{
+		Nickname: nickname,
+	})
+
+	if err != nil {
+		return formatError(err)
+	}
+
+	if resp == nil {
+		return errors.New(AccountInternalError)
+	}
+	switch resp := resp.Response.(type) {
+	case *pb.RenamePeerResponse_ServiceErrorCode:
+		return serviceErrorCodeToError(resp.ServiceErrorCode)
+
+	case *pb.RenamePeerResponse_UpdatePeerErrorCode:
+		return errors.New(AccountInternalError)
+	case *pb.RenamePeerResponse_MeshnetErrorCode:
+		return meshnetErrorToError(resp.MeshnetErrorCode)
+
+	case *pb.RenamePeerResponse_RenamePeerErrorCode:
+		switch resp.RenamePeerErrorCode {
+		case pb.RenamePeerErrorCode_SAME_NICKNAME:
+			return fmt.Errorf(MsgMeshnetPeerSetNicknameTheSame, nickname)
+		}
+	}
+
+	color.Green(MsgMeshnetSetNicknameSuccessful, nickname)
+	return nil
+}
+
+func (c *cmd) MeshRemoveMachineNickname(ctx *cli.Context) error {
+	resp, err := c.meshClient.RemoveMachineNickname(context.Background(), &pb.ChangeMachineNicknameRequest{
```

Comment:
Why not call `ChangeMachineNickname` RPC directly?

Reply:
probably leftovers :). I'll remove it.